### ISSUE TITLE
works by unspecified authors

### ISSUE THREAD

**Issue by LenaHenningsen** [readchina/ReadingData#355]
Do we have a way to include works into the database that are somewhat unspecified, i.e. if someone says all the books she like to read where a) banned books, or b) by Russian authors?

**Comment by duncdrum** [comment]
We have structural entries for unknown author, unknown title, etc. But the questions is in how far  are you still recording a reading act if everything about it is unknown. Less philosophical and more practical: how much use would an act be in which everything is unknown?

**Comment by LenaHenningsen** [comment]
… true – though I find the info meaningful…

 

On a different note – Travis told me 
that there is something wrong in files I did not even touch… (art work, membership, quotation, social relations…) and 
something about inconsistent values for start, end, and page number which I don’t understand
in the persons file, he apparently does not like people born before year 0 and has an issue with some source IDs…
similarly for the primary and secondary source files
zenmeban?
 

(It did however point out to me two act numbers that I gave wrongly, so at least this I did understand…)

Lena

 

Von: Duncan Paterson <notifications@github.com>
Antworten an: readchina/ReadingData <reply@reply.github.com>
Datum: Montag, 18. Februar 2019 um 21:08
An: readchina/ReadingData <ReadingData@noreply.github.com>
Cc: LenaHenningsen <Lena.Henningsen@sinologie.uni-freiburg.de>, Author <author@noreply.github.com>
Betreff: Re: [readchina/ReadingData] works by unspecified authors (#355)

 

We have structural entries for unknown author, unknown title, etc. But the questions is in how far are you still recording a reading act if everything about it is unknown. Less philosophical and more practical: how much use would an act be in which everything is unknown?
